examples/integration-tutorials/using-the-gmail-connector/guide/health_care_service.bal

```diff
@@ -96,9 +96,12 @@ service hospitalMgtService on new http:Listener(9092) {
                 }
                 // call payment settlement
                 http:Response paymentResponse = doPayment(untaint responsePayload);
-                // send the response back to the client
-                //respondToClient(caller, paymentResponse);
-                respondToClient(caller, sendEmail(generateEmail(untaint paymentResponse)));
+            // send the response back to the client
```

Comment:
Shall we add the relevant issue link as a comment here?

Reply:
Done.